content/en/docs/gke/pipelines/upgrade.md

```diff
@@ -1,134 +1,55 @@
+| full Kubeflow (>=1.1)                   | ✅                | ✅                                  | Needs documentation                   | ✅                                                                                  |
+| full Kubeflow (<1.1)                    |                  | ✅                                  | ✅                                     |                                                                                    |
+| AI Platform Pipelines                   |                  | ✅                                  |                                       |                                                                                    |
+| AI Platform Pipelines (managed storage) |                  | ✅                                  | ✅                                     |                                                                                    |
 
 Notes:
+* Managed storage means storing all the data in managed storage services on Google Cloud. A Google Cloud Storage (GCS) bucket and a Cloud SQL instance is required. Using managed storages make it easy to manage, back up, and restore Kubeflow Pipelines data.
 
-* You must use command-line deployment. 
-  You cannot reinstall Kubeflow Pipelines using the web interface.
-* When you do `kfctl apply` or `kfctl build`, you should use a different 
-  deployment name from your existing deployment name. Otherwise, kfctl will
-  delete your data in the existing PDs. This guide defines the deployment name
-  in the ${KF_NAME} environment variable. 
+## Kubeflow Pipelines Standalone
 
-To reinstall Kubeflow Pipelines:
+Upgrade Support for Kubeflow Pipelines standalone is **Beta**.
 
-1. Follow the [command-line deployment 
-  instructions](/docs/gke/deploy/deploy-cli/), but **note the following
-  changes in the procedure**.
+[Upgrading Kubeflow Pipelines Standalone](/docs/pipelines/installation/standalone-deployment/#upgrading-kubeflow-pipelines) introduces how to upgrade in place.
 
-1. Set a different `${KF_NAME}` name from your existing `${KF_NAME}`.
+## Full Kubeflow
 
-1. **Before** running the `kfctl apply` command:
+On Google Cloud, the full Kubeflow follows [the blueprint pattern](https://googlecontainertools.github.io/kpt/guides/producer/blueprint/) starting from Kubeflow 1.1.
 
-    * Edit `${KF_DIR}/gcp_config/storage-kubeflow.yaml` and set the following
-      flag to skip creating new storage:
+The blueprint pattern makes it easy to upgrade the full Kubeflow in-place, refer to [Update Kubeflow on Google Cloud](docs/gke/deploy/deploy-cli/#update-kubeflow) documentation for instructions.
 
-      ```
-      ...
-      createPipelinePersistentStorage: false
-      ...
-      ```
+However, there's no current support to upgrade from Kubeflow 1.0 or earlier to Kubeflow 1.1 whiling keeping Kubeflow Pipelines data. Provide your feedback in [kubeflow/pipelines#4346](https://github.com/kubeflow/pipelines/issues/4346) if it's important to you.
```

Comment:
Thanks for the suggestions!
I didn't add the "as of the date of this guide" part, because if that every happens to change, we will update the doc right away, so I don't think outdated doc is an issue.